features/bevragen/persoon/nationaliteit/dev/fields-gba.feature

```diff
@@ -0,0 +1,167 @@
+# language: nl
+
+Functionaliteit: GBA Persoon: Nationaliteit
+
+Rule: wanneer één of meerdere velden van een nationaliteit wordt gevraagd, dan wordt ook de waarde van 'nationaliteit (05.10)' geleverd
+
+  Abstract Scenario: alle velden van een nationaliteit wordt gevraagd met field pad 'nationaliteiten'
+    Gegeven de persoon met burgerservicenummer '000000012' heeft een 'nationaliteit' met de volgende gegevens
+    | naam                  | waarde |
+    | nationaliteit (05.10) | <code> |
+    | reden opname (63.10)  | 001    |
+    Als gba personen wordt gezocht met de volgende parameters
+    | naam                | waarde                          |
+    | type                | RaadpleegMetBurgerservicenummer |
+    | burgerservicenummer | 000000012                       |
+    | fields              | nationaliteiten                 |
+    Dan heeft de response een persoon met een 'nationaliteit' met alleen de volgende gegevens
+    | naam                       | waarde                                            |
+    | nationaliteit.code         | <code>                                            |
```

Comment:
ik dacht dat we bij nationaliteit (05.10)==0000 (Onbekend) het veld type met waarde "NationaliteitOnbekend" leveren. Daarin zit geen veld nationaliteit.

Moeten de specificaties nog wijzigen waarbij we de polymorfie op nationaliteit laten vallen?